Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -48,19 +48,18 @@ struct FloatingPanel<Content>: View where Content: View {
     /// The height of the content.
     @State private var height: CGFloat = .minHeight
     
-    /// A Boolean value indicating whether the keyboard is open.
-    @State private var keyboardIsOpen = false
+    /// The current height of the device keyboard.
+    @State private var keyboardHeight: CGFloat = 0
```

Comment:
Tracking the height allows us to apply the correct amount of padding to implement manual keyboard avoidance. This height can differ slightly depending on whether the keyboard has a toolbar attached or the predictive text bar is enabled.